GsaGH/Parameters/5_Results/VectorResultGoo.cs

```diff
@@ -12,9 +12,10 @@ namespace GsaGH.Parameters
   /// </summary>
   public class VectorResultGoo : GH_GeometricGoo<GH_Vector>, IGH_PreviewData
   {
-    public readonly Point3d StartingPoint;
+    public Point3d StartingPoint;
     public Vector3d Direction { get; private set; }
-    public readonly IQuantity ForceValue;
+    public IQuantity ForceValue;
```

Comment:
why do we want to change it outide the class?

Reply:
users might use the scripting component to access the result IQuantity values